Automattic/newspack-app-shell
# Configure JS linting and formatting

This configures JS code linting and formatting, using `eslint` and `prettier`, respectively. 

## How to test it

1. run `npm run lint` - it should finish without errors
1. run `npm run format` - no files should be changed
